Assets/Scripts/PlayerController.cs

```diff
@@ -8,6 +8,7 @@ public class PlayerController : MonoBehaviour
     [SerializeField]
     private const float MoveSpeed = 2f;
     private const float JumpForce = 5.2f;
+    private const float DistToGround = 1.0f;
```

Comment:
is this the distance from player transform to ground?

Reply:
from the draggy thingy to the ground, yes.